### PR TITLE
add fallback logic for agendas with no sort date to sort

### PR DESCRIPTION
**Pull request by M00staff**
[Ticket](linkToTicket)

### CODE REVIEW

File: talentmap_api/common/common_helpers.py [hunk 1]
```diff
@@ -501,9 +501,26 @@ def sort_legs(agendaLegs):
...
+
+    # filter out the nulls
+    # sort by ETA then by TED
+    nullLegs = []
+    for idx, val in enumerate(agendaLegs):
+        if not val['sort_date']:
+            if val['eta']:
+                val['sort_date'] = val['eta']
+            elif val['ted']:
+                val['sort_date'] = val['ted']
```
Comment (scott062) [inline talentmap_api/common/common_helpers.py:513]: Move this logic into the sort_date mapping

sort_date = TED if Sep else ETA else None

Comment (scott062) [reply talentmap_api/common/common_helpers.py:513]: We can handle the nulls as this legacy func suggests by putting them at end but eventually will get better rules to update

Comment (M00staff) [reply talentmap_api/common/common_helpers.py:513]: isn't that what this does?
image

Comment (M00staff) [reply talentmap_api/common/common_helpers.py:513]: oh - nevermind - i see

Comment (M00staff) [reply talentmap_api/common/common_helpers.py:513]: something wonky with this logic

image

if i force pass an Ai with `None` as the `sort_date` it doesn't get popped out of the list 

image

image

**Review by scott062 — APPROVED** [review]

**Pull request merged**